File: tests/fixtures/doc_comments.cpp
/**
 * A documented function.
 * @param name the name
 * @return greeting string
 */
std::string greet(const std::string& name) {
    return "Hello, " + name;
}

// Regular comment, not Doxygen.
void noDoc() {}

/**
 * Documented class.
 */
class Widget {
public:
    /** Constructor doc. */
    Widget(int id) : id_(id) {}
private:
    int id_;
};

File: tests/fixtures/DocComments.cs
/// <summary>
/// A documented class.
/// </summary>
public class DocComments {
    /// <summary>
    /// XML doc on a method.
    /// </summary>
    /// <param name="name">The name.</param>
    public string Greet(string name) {
        return "Hello, " + name;
    }

    // Regular comment.
    public void NoDoc() {}
}
